src/map/packets_struct.h

```diff
@@ -256,6 +291,12 @@ struct NORMALITEM_INFO {
 #endif
 } __attribute__((packed));
 
+struct RndOptions {
+	short index;
+	short value;
+	unsigned char param;
+};
```

Comment:
It's a good idea to use the `__attribute((packed))` in this one as well, to keep it consistent with all compilers (i.e. the ones that don't understand the `#pragma pack` at line 256).